src/test/resources/org/apache/sling/feature/cpconverter/handlers/i18n-jsonfile-xml-descriptor-test/en.json

```diff
@@ -0,0 +1,42 @@
+{
+  "com.schindler.aem.components.TeaserImpl.fallbackaction": "Learn more",
```

Comment:
i would recommend to avoid using AEM customer information in an apache project. please refactor the test content.